scoop-update-notifier.Tests.ps1

```diff
@@ -0,0 +1,44 @@
+BeforeAll {
+    # NOTE: Import a function from a self contained script without execution.
+    # This magic from https://jakubjares.com/2019/06/09/2019-07-testing-whole-scripts/ .
+    function f () {}
+    New-Alias -Name Notify-Update -Value f
+    . $PSScriptRoot/scoop-update-notifier.ps1 
+    Remove-Item Alias:Notify-Update
+    Remove-Item "function:/f"
+
+    if (Test-Path .\previous-updates) {
```

Comment:
In scoop-update-notifier.ps1, the file name is defined as string. Unify way of definition.

Reply:
Applied in https://github.com/ekishouTV/scoop-update-notifier/pull/1/commits/37604e1b55bf39ba78a77572b2d742c0530eec18